### ISSUE TITLE
Curly brackets not rendered when using .latex('\\{') in combination with fade-in

### ISSUE THREAD

**Issue by sjcjoosten** [mathquill/mathquill#990]
When I run:

```
mq.latex('\\left\\{1,\\right\\}');
```

I expect to see {1,}.

The LaTeX, spacing and characters all seem to be there, but the { and } seem transparent or hidden. The 1, is shown.

**Comment by jaltekruse** [comment]
Hi @sjcjoosten,

I tried reproducing this and I'm not seeing the described behavior, can you send a repro video or more details?


https://user-images.githubusercontent.com/365366/184935661-a033b661-9ee9-41a6-ab2b-d8de7553ebf1.mp4

**Comment by sjcjoosten** [comment]
Sorry for not testing this in isolation. I figured out what the problem was, running these two examples from the console on the MQ homepage:

```Javascript
d = document.createElement('div');mf=MQ.MathField(d,{handlers:{enter:()=>{}}});$('#example').append(d);d.style='display:none';$(d).fadeIn(200);mf.latex('\\left\\{1,\\right\\}');

d = document.createElement('div');mf=MQ.MathField(d,{handlers:{enter:()=>{}}});$('#example').append(d);d.style='display:none';mf.latex('\\left\\{1,\\right\\}');$(d).fadeIn(200);
```

First of these works like a charm, the second has the exact same issue I'm facing. This immediately gives me a work-around (i.e. do the first thing and not the second) but it would be nice to be able to have both of these work.

**Comment by jaltekruse** [comment]
Oh in that case I have good news. There is a branch that we are finishing reviewing with changes made by the Desmos team on their fork. One of the changes was a performance improvement to how "strechy symbols" like curly braces, parenthesis, etc. are scaled and displayed. Their approach looks to have fixed this behavior after a quick test on top of their branch.

The mathquill PR for merging this in:
https://github.com/mathquill/mathquill/pull/947

Original PR on the desmos fork that introduced the feature:
https://github.com/desmosinc/mathquill/pull/107

**Comment by sjcjoosten** [comment]
Will be fixed by #947
